userspace/falco/falco_outputs.cpp

```diff
@@ -77,9 +79,12 @@ void falco_outputs::init(bool json_output,
 	m_time_format_iso_8601 = time_format_iso_8601;
 	m_hostname = hostname;
 
+	m_worker_thread = std::thread(&falco_outputs::worker, this);
```

Comment:
Since `falco_outputs::worker` is not supposed to throw exceptions (at least, that's the intention), I'd rather add `noexcept` to it.

I just need to think a bit about that. I'm still open to any further suggestions!

Reply:
@leogr that would not be sufficient if `falco_outputs::worker` calls other functions without `noexcept` guarantee

Reply:
Hey @deepskyblue86 

Although your point is totally correct, in the end, @leodido and I (as discussed privately) opted for `noexcept` since the ultimate goal is to have no exceptions within the worker. However, as this is not completely the case, I added a comment in the code explaining that it needs to be improved (see https://github.com/falcosecurity/falco/pull/1451/commits/cf38e6351c49fbc4cb0d564f045cb8acea849e13).
Moreover, other similar issues around in the codebase probably need to be addressed in bulk in a different PR.

Thank you for having spotted that anyway!